01-introduction.Rmd

```diff
@@ -0,0 +1,204 @@
+
+To open RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Open__ button to the right of the tool's name.
+
+## Restart RStudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow is not working as expected, it may help to restart it.
+
+To restart RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Restart__ button to the right of the tool's name.
+
+## Unidle Rstudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow instance is inactive for an extended period of time (e.g., overnight) it will be idled.
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Unidle__ button to the right of the tool's name. 
+
+Unidling usually only takes a few seconds, however, it can take up to several minutes.
+
+## Configure Git and GitHub
+
+To configure Git and GitHub for the Analytical Platform, you must complete the following steps:
+
+1.  [Create an SSH key](#create-an-ssh-key).
+2.  [Add the SSH key to GitHub](#add-the-ssh-key-to-github).
+3.  [Configure your username and email in Git on the Analytical Platform](#configure-your-username-and-email-in-git-on-the-analytical-platform).
+
+### Create an SSH key
+
+You can create an SSH key in RStudio or JupyterLab. We recommend that you use RStudio.
+
+#### RStudio
+
+To create an SSH key in RStudio, follow the steps below:
+
+1.  Open RStudio from the Analytical Platform control panel.
+2.  In the menu bar, select __Tools__ then __Global Options...__
+3.  In the options window, select __Git/SVN__ in the navigation menu.
+4.  Select __Create RSA key...__
+5.  Select __Create__.
+6.  Select __Close__ when the information window appears.
+7.  Select __View public key__.
+8.  Copy the SSH key to the clipboard by pressing Ctrl+C on Windows or ⌘C on Mac.
+
+#### JupyterLab
+
+To create an SSH key in JupyterLab, follow the steps below:
+
+1.  Open JupyerLab from the Analytical Platform control panel.
+2.  Select the __+__ icon in the file browser to open a new Launcher tab.
+3.  Select __Terminal__ from the 'Other' section.
+4.  Create an SSH by running:
+    ```
+    ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+    ```
+     Here, you should substitute the email address you used to sign up to GitHub.
+5.  When prompted to enter a file in which to save the key, press Enter to accept the default location.
+6.  When prompted to enter a passphrase, press Enter to not set a passphrase.
+6.  View the SSH key by running:  
+    ```
+    cat /home/jovyan/.ssh/id_rsa.pub
+    ```
+7.  Select the SSH key and copy it to the clipboard by pressing Ctrl+C on windows or ⌘C on Mac.
```

Comment:
```suggestion
7.  Select this SSH public key and copy it to the clipboard by pressing Ctrl+C on windows or ⌘C on Mac.
```

Reply:
As [here](https://github.com/moj-analytical-services/platform_user_guidance/pull/92#discussion_r285179970), I'm not sure whether it is necessary and improves clarity to say 'public'.